ros2param/test/test_verb_load.py

```diff
@@ -344,8 +344,13 @@ def test_verb_load_wildcard(self):
             ) as param_dump_command:
                 assert param_dump_command.wait_for_shutdown(timeout=TEST_TIMEOUT)
             assert param_dump_command.exit_code == launch_testing.asserts.EXIT_OK
-            loaded_params = yaml.safe_load(param_dump_command.output)
-            params = loaded_params[f'{TEST_NAMESPACE}/{TEST_NODE}']['ros__parameters']
+            try:
+                loaded_params = yaml.safe_load(param_dump_command.output)
```

Comment:
in the case of failure, this could return string but dictionary. if that happens it will lead the TypeError exception. this PR checks that condition, and fails if something went wrong with return value.